client/windows/CCastleInterface.h

```diff
@@ -45,9 +45,11 @@ class CBuildingRect : public CShowableAnim
 	enum EBuildingCreationAnimationPhases : uint32_t
 	{
 		BUILDING_APPEAR_TIMEPOINT = 500, //500 msec building appears: 0->100% transparency
-		BUILDING_WHITE_BORDER_TIMEPOINT = 1000, //500 msec border glows from white to yellow
-		BUILDING_YELLOW_BORDER_TIMEPOINT = 1500, //500 msec border glows from yellow to normal
-		BUILD_ANIMATION_FINISHED_TIMEPOINT = 2500 //1000 msec delay, nothing happens
+		BUILDING_WHITE_BORDER_TIMEPOINT = 900, //400 msec border glows from white to yellow
+		BUILDING_YELLOW_BORDER_TIMEPOINT = 1100, //200 msec border glows from yellow to normal (dark orange)

```

Comment:
BTW take into account hd mod or hota (forgot which one) speeds up the glow - I am unsure if you used base H3 as reference

Reply:
also for me on linux H3 with HD seems to be missing stuff like black screen fade out when entering towns and battles etc - can test without HD later.

Reply:
And here I thought that I disabled all "enhancements" from HD Mod.
Yeah, I've noticed that this animation feels "wrong" but thought that this is just H3 behavior. 
Will recheck.

Reply:
I would keep separate comment for HD mod timing - may be useful for future use

Reply:
Fixed. All other timings look OK, for battles will use vanilla game timings. Have even less trust in HD mod timings in battles